dash_bio/component_factory/_volcano.py

```diff
@@ -75,10 +76,21 @@ def VolcanoPlot(
 - title (string; default 'Volcano Plot'): Title of the graph.
 - xlabel (string; optional): Label of the x axis.
 - ylabel (string; default '-log10(p)'): Label of the y axis.
+- xaxis (dict: optional): Dict with compatible properties for the x-axis of
+     the figure layout.
+- yaxis (dict: optional): Dict with compatible properties for the y-axis of
+    the figure layout.
```

Comment:
Let's move these (and height, width, margin, legend below) into the "Additional keys" section of the docstring, saying something like "Some commonly-used keys are:" and then exactly the text you have currently. These keys don't explicitly show up in the function signature so it'll be confusing to have them mixed in like this.

Reply:
Updated in e3dfcb4.